docs/berkeley-upgrade/migrating-archive-database-to-berkeley.mdx

```diff
@@ -257,20 +406,14 @@ In the second phase of migration, add property `target_epoch_ledgers_state_hash`
 }
 ```
 
-#### How to start replayer migration from latest checkpoint
+#### I am receiving errors from gsutil regarding missing precomputed blocks
 
-Instead of using Mainnet ledger as input config, provide the checkpoint for the `--input-config` parameter.
+When migrating last chunk of blocks from mainnet, there can be an issue for runs with
+large `batch-size` argument inputs. As an optimization gsutil is trying to fetch as `block-size` amount of precomputed blocks.
+However, when precomputed blocks bucket has not contain enough (= `block-size` amount) precomputed blocks gsutil may panic stopping migration process.
+Our recommendation is to restart process with smaller number of `batch-size`. If you are running migration just after fork block is announced it is best
+to have `--block-size` argument set up to 5.
 
-### Migration app parameters reference
-
-For more advanced users and operators, all the parameters that can influence the migration process are listed here:
-
-##### Berkeley migration app
-
-- **batch-size NN**  - is used when downloading precomputed blocks. The bigger the buffer, the more precomputed blocks are downloaded in a single fetch.
-- **end-global-slot NN (Optional)** - Last global slot since genesis to include in the migration (if omitted, the default is to migrate only canonical blocks).
-
-##### Replayer in migration mode
+#### How to start replayer migration from latest checkpoint
 
-- **checkpoint-interval NN** - Write a checkpoint file for every NN slot.
-- **continue-on-error** - Continue processing after errors
+Instead of using Mainnet ledger as input config, provide the checkpoint for the `--input-config` parameter.
```

Comment:
we'll need to use consistently the same Markdown syntax and capitalization for "Mainnet ledger"